zephyr/lib/cpu.c

```diff
@@ -63,6 +63,7 @@ static FUNC_NORETURN void secondary_init(void *arg)
 #if CONFIG_ZEPHYR_NATIVE_DRIVERS
 #include <sof/trace/trace.h>
 #include <rtos/wait.h>
+#include <intel_adsp_ipc.h>
```

Comment:
This needs to be wrapped into ifdef CONTEXT_SAVE. This is in general a bit iffy to have Intel specific code here when this code should be generic for all vendors. But alas we have the context save code already in this file, but let's at least use that so you can compile this file for other platforms.